example/templates/person.ts

```diff
@@ -10,10 +10,14 @@ const schema = {
       'type': 'object',
       'properties': {
           'age': {
-              'type': 'integer'
+              'type': 'integer',
+              'maximum': 25,
```

Comment:
I'm not a huge fan of adding the minimum and maximum to the `age` and `height` properties. I would prefer to have either
- a dedicated example for numbers
or 
- add additional properties for the slider control to the person schema

but this is not a blocker in my opinion

Reply:
yes actually I want to add another property called`drivingSkill`